packages/driver/src/cypress.ts

```diff
@@ -56,8 +63,113 @@ const throwPrivateCommandInterface = (method) => {
   })
 }
 
+interface BackendError extends Error {
+  __stackCleaned__: boolean
+  backend: boolean
+}
+
+interface AutomationError extends Error {
+  automation: boolean
+}
+
 class $Cypress {
+  cy: any
+  chai: any
+  mocha: any
+  runner: any
+  downloads: any
+  Commands: any
+  $autIframe: any
+  onSpecReady: any
+  events: any
+  $: any
+  arch: any
+  spec: any
+  version: any
+  browser: any
+  platform: any
+  testingType: any
+  state: any
+  originalConfig: any
+  config: any
+  env: any
+  getTestRetries: any
+  Cookies: any
+  ProxyLogging: any
+  _onInitialize: any
+  isCy: any
+  log: any
+  isBrowser: any
+  emit: any
+  emitThen: any
+  emitMap: any
+
+  $Cypress: any
+  Cy: any
+  Chainer: any
+  Command: any
+  dom: any
+  errorMessages: any
+  Keyboard: any
+  Location: any
+  Log: any
+  LocalStorage: any
+  Mocha: any
+  resolveWindowReference: any
+  resolveLocationReference: any
+  Mouse: any
+  Runner: any
+  Server: any
+  Screenshot: any
```

Comment:
for all these declarations that are assigned in `constructor`, why not just do this?

```suggestion
  Screenshot = $Screenshot
```

then you wouldn't need to assign `this.Screenshot = $Screenshot` later on, and the types would automatically be updated when `$Screenshot` is typed

Reply:
I changed it that way. But I had to remove `Cookie` and `Commands`, because they're reassigned in the code below.